apps/web/ee/components/team/availability/TeamAvailabilityModal.tsx

```diff
@@ -32,62 +34,66 @@ export default function TeamAvailabilityModal(props: Props) {
 
   return (
-    <div className="flex max-h-[500px] min-h-[500px] flex-row  space-x-8 rtl:space-x-reverse">
-      <div className="min-w-64 w-64 space-y-5 p-5 pr-0">
-        <div className="flex">
-          <Avatar
-            imageSrc={getPlaceholderAvatar(props.member?.avatar, props.member?.name as string)}
-            alt={props.member?.name || ""}
-            className="h-14 w-14 rounded-full"
-          />
-          <div className="ml-3 inline-block pt-1">
-            <span className="text-lg font-bold text-neutral-700">{props.member?.name}</span>
-            <span className="-mt-1 block text-sm text-gray-400">{props.member?.email}</span>
+    <LicenseRequired>
+      <div className="flex max-h-[500px] min-h-[500px] flex-row  space-x-8 rtl:space-x-reverse">
+        <div className="min-w-64 w-64 space-y-5 p-5 pr-0">
+          <div className="flex">
+            <Avatar
+              imageSrc={getPlaceholderAvatar(props.member?.avatar, props.member?.name as string)}
+              alt={props.member?.name || ""}
+              className="h-14 w-14 rounded-full"
+            />
+            <div className="ml-3 inline-block pt-1">
+              <span className="text-lg font-bold text-neutral-700">{props.member?.name}</span>
+              <span className="-mt-1 block text-sm text-gray-400">{props.member?.email}</span>
+            </div>
+          </div>
+          <div className="flex flex-col">
+            <span className="font-bold text-gray-600">Date</span>
+            <DatePicker
+              date={selectedDate.toDate()}
+              onDatesChange={(newDate) => {
+                setSelectedDate(dayjs(newDate));
+              }}
+            />
+          </div>
+          <div>
+            <span className="font-bold text-gray-600">Timezone</span>
+            <TimezoneSelect
+              id="timeZone"
+              value={selectedTimeZone}
+              onChange={(timezone) => setSelectedTimeZone(timezone.value)}
+              classNamePrefix="react-select"
+              className="react-select-container mt-1 block w-full rounded-sm border border-gray-300 shadow-sm focus:border-neutral-800 focus:ring-neutral-800 sm:text-sm"
+            />
+          </div>
+          <div>
+            <span className="font-bold text-gray-600">Slot Length</span>
+            <Select
+              options={[
+                { value: 15, label: "15 minutes" },
+                { value: 30, label: "30 minutes" },
+                { value: 60, label: "60 minutes" },
+              ]}
+              isSearchable={false}
+              classNamePrefix="react-select"
+              className="react-select-container focus:ring-primary-500 focus:border-primary-500 block w-full min-w-0 flex-1 rounded-sm border border-gray-300 sm:text-sm"
+              value={{ value: frequency, label: `${frequency} minutes` }}
+              onChange={(newFrequency) => setFrequency(newFrequency?.value ?? 30)}
```

Comment:
define this as `const defaultFrequency = 30;` and use it here

Reply:
This is not my change. It cane from upstream it seems